test/e2e/monitor.go

```diff
@@ -23,7 +23,9 @@ var _ = Describe("Monitor", func() {
 		wg.Add(1)
 		mon, err := cluster.NewMonitor(log, clients.RestConfig, &api.OpenShiftCluster{
 			ID: resourceIDFromEnv(),
-		}, &noop.Noop{}, nil, true, &wg)
+		}, &api.OpenShiftClusterDocument{
```

Comment:
I think it might be worth adding E2E tests for both the monitor and Geneva Actions functionality. We have various contexts in which E2E runs against an RP with Hive enabled (production/release E2E, PR E2E after we move to the containerized implementation).

Reply:
If it is too difficult to add the E2E tests in this PR, this can become follow-up work, however.

Reply:
New story created for the same